### PR TITLE
Add even more swedish translations :)

### PR DESCRIPTION
**Pull request by alvesvaren**
# What does it do?

Adds swedish translations to more parts of the application

# Why is it needed?

I've used the old translations for a while and want to improve some of them, and I've added new translations for some parts that were not translated.

# How to test it?

Build the application and set the admin interface language to "sv", and check so that everything is correctly translated

### CODE REVIEW

File: packages/core/admin/admin/src/translations/sv.json [hunk 1]
```diff
@@ -79,52 +79,70 @@
...
+  "Settings.apiTokens.duration.30-days": "30 days",
+  "Settings.apiTokens.duration.90-days": "90 days",
+  "Settings.apiTokens.duration.unlimited": "Unlimited",
+  "Settings.apiTokens.form.duration": "Token duration",
+  "Settings.apiTokens.form.type": "Token type",
+  "Settings.apiTokens.duration.expiration-date": "Expiration date",
+  "Settings.apiTokens.createPage.permissions.title": "Permissions",
+  "Settings.apiTokens.createPage.permissions.description": "Only actions bound by a route are listed below.",
+  "Settings.apiTokens.RegenerateDialog.title": "Regenerate token",
+  "Settings.apiTokens.popUpWarning.message": "Are you sure you want to regenerate this token?",
```
Comment (ronronscelestes) [inline packages/core/admin/admin/src/translations/sv.json:109]: out of curiosity, some translations like this one seem to be in English - is it intended or is it an oversight? 🙂

**Review by ronronscelestes — COMMENTED** [review]
hi @alvesvaren, that's a lot of great addition, thank you!

File: packages/core/admin/admin/src/translations/sv.json (hunk 1 above)
Comment (alvesvaren) [reply packages/core/admin/admin/src/translations/sv.json:109]: No that's not intended, will fix :+1:

Comment (alvesvaren) [reply packages/core/admin/admin/src/translations/sv.json]: Fixed as of c369330

**Review by ronronscelestes — APPROVED** [review]
LGTM thank you!

**Pull request merged**